### PR TITLE
Add 'OCP\Files\IMimeTypeDetector' to DI container

### PR DESCRIPTION
**Pull request by rullzer**
We still missed that see https://github.com/owncloud/mail/issues/1207

Easy one to review :)

CC: @MorrisJobke @nickvergessen @PVince81 @LukasReschke @ChristophWurst

### CODE REVIEW

**Comment by MorrisJobke** [comment]
:+1:

**Comment by ChristophWurst** [comment]
:+1: thanks

File: tests/lib/server.php [hunk 1]
```diff
@@ -94,6 +94,8 @@ public function dataTestQuery() {
 			['HttpClientService', '\OCP\Http\Client\IClientService'],
 
 			['IniWrapper', '\bantu\IniGetWrapper\IniGetWrapper'],
+			['MimeTypeDetector', '\OCP\Files\IMimeTypeDetector'],
+			['MimeTypeDetector', '\OC\Files\Type\Detection'],
```
Comment (nickvergessen) [inline tests/lib/server.php]: the alphabet got messed up in nl? :palm_tree:

**Comment by nickvergessen** [comment]
:+1: thou

File: tests/lib/server.php (hunk 1 above)
Comment (rullzer) [inline tests/lib/server.php]: mmmm no... just not paying attention I think :S

**Pull request merged**